test/db.test.js

```diff
@@ -0,0 +1,55 @@
+require('./../config/config');
+
+const mongoose = require('./../server/db/mongoose');
+const expect = require('expect');
+
+const { Repository } = require('./../server/db/repositories');
+const { Username } = require('./../server/db/username');
+const { User } = require('./../server/db/user');
+
+// console.log(mongoose.connection.host);
```

Comment:
This code should go away. No commented code here unless there's a super strong reason 👍